src/js/components/RangeSelector/EdgeControl.js

```diff
@@ -48,22 +48,51 @@ class EdgeControl extends Component {
         theme.rangeSelector.edge &&
         theme.rangeSelector.edge.type) ||
       'disc';
+    let node;
+    if (type === 'bar') {
+      node = (
+        <Box
+          flex
+          justifySelf="stretch"
+          width={`${size}px`}
+          background={normalizeColor(color || 'control', theme)}
+          border={
+            focused ? { color: normalizeColor('focus', theme) } : undefined
+          }
```

Comment:
I think we also need `style` with `extend` for the bar type.

Reply:
same note as below 👇